### ISSUE TITLE
Location arrow stays solid after few days of use

### ISSUE THREAD

**Issue by GeVaSta** [LocativeHQ/Locative-iOS#50]
After adding the first geofence, the location arrow (right top of iphone 6s) Is solif for a few seconds and then turns to hollow (iphone is in geofence state).
After some days of usage however, after unlocking the phone, the arrow stays solid. It is an error I can reproduce. After erasing the geofence, the arrow stays on. After reinstalling the app, it works for a few days as it shoud be (hollow arrow after few seconds).

**Comment by kimar** [comment]
Sometimes Locative uses GPS (e.g. when dispatching HTTP webhooks), the icon just indicates when / if GPS or Geofencing has been used. 
This is not a bug and the arrow is no indicator if an app drains your battery by e.g. using GPS, if that's your concern please refer to the battery usage section and see that Locative isn't using much battery.

![screenshot](https://cloud.githubusercontent.com/assets/126418/17642890/6206701a-619b-11e6-9ddf-35618a7fb3da.jpg)

**Comment by GeVaSta** [comment]
I don't agree with your explanation. When Locative uses GeoFencing (when in the background or even after been killed), it should not 'light' the solid Location arrow in the top titlebar of the iPhone. It should only light the hollow arrow meaning it is Geofencing. When the app starts or the iphone is unlocked, it is okay for a solid arrow for short time but after closing the app or leaving it alone, it should resume to hollow.
This way it works for a few days but at a certain moment it stays solid; this is not normal behaviour.
This way, besides the fact that perhaps it does not generate extra battery consumption, I am unable to track other apps for Location usage because the Locative app already keeps the arrow on.

**Comment by kimar** [comment]
I'm reopening this in case anyone want's to look into it. I don't see a way to change this here right now as those arrows are controlled by iOS.
